internal/resources/metal/project/sweeper.go

```diff
@@ -0,0 +1,44 @@
+package project
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/equinix/terraform-provider-equinix/internal/sweep"
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+)
+
+func AddTestSweeper() {
+	resource.AddTestSweepers("equinix_metal_project", &resource.Sweeper{
+		Name:         "equinix_metal_project",
+		Dependencies: []string{"equinix_metal_vlan"},
+		F:            testSweepProjects,
+	})
+}
+
+func testSweepProjects(region string) error {
+	log.Printf("[DEBUG] Sweeping projects")
+	config, err := sweep.GetConfigForMetal()
+	if err != nil {
+		return fmt.Errorf("[INFO][SWEEPER_LOG] Error getting configuration for sweeping projects: %s", err)
```

Comment:
In the AWS provider, errors from the sweeper don't have the `[INFO][SWEEPER_LOG]` prefixes, they have no prefix. Just calling this out, I don't know what is "right".

Reply:
A few lines later we return an error with no prefix.